travis_specific/tests.sh

```diff
@@ -5,3 +5,7 @@ cd resources/tests/endToEndTests
 bash IntContainerTest.sh
 bash MultipleConstructorsTest.sh
 bash TransactionInConstructorTest.sh
+
+cd ../YulTests
```

Comment:
Add comments indicating that the tests above are for Fabric, and the tests above are for Yul.

Reply:
Also, really, the Fabric tests should now be moved into a subdirectory, FabricTests.